styles.css

```diff
@@ -111,3 +111,105 @@
 .main__button:hover {
   opacity: .5;
 }
+
+@media (max-width: 1560px) {
+  .wrapper {
+    width: 69%;
+}
+
+.menu__link {
+  font: 500 14px/16px 'Roboto', sans-serif, arial, helvetica;
+}
+
+  .main {
+  padding: 0 58% 100px 0;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html-css-2-0/pullRequest?prid=6306727)